test/functional/tools/validation_image_metadata.xml

```diff
@@ -0,0 +1,176 @@
+<tool id="validation_image_metadata" name="validation_image_metadata" version="1.0">
```

Comment:
Would a unit test (maybe analogous to https://github.com/galaxyproject/galaxy/blob/ec6831e14b7742208d9e5bf7752cbb691e1fabd1/test/unit/data/datatypes/test_qiime2.py#L74) be favorable? At least it would be faster.

Reply:
Thanks for the hint!

Fixed this in 6e7efb9399bf57579df49c5abb6ce01c266c3c72

Will remove the functional tests next.